docs/mathematics/happy-number.mdx

```diff
@@ -0,0 +1,244 @@
+---
+title: Happy Number
+slug: /mathematics/happy-number
+---
+
+## Introduction
+
+A number is called a happy number if it converges to 1 after a sequential operation wherein each step we replace the original number by the sum of the squares of its individual digits.When we have a digit given to us, we perform the operation, the sum of squares of its individual digits but in some cases, we might arrive at the same number which we already came across. In such cases, we can clearly see that it would never terminate so such numbers are not considered as happy numbers.
+
```

Comment:
Replace this: `When we have a digit given to us, we perform the operation, the sum of squares of its individual digits but in some cases, we might arrive at the same number which we already came across.`

with: 

`When a digit is inputted, we calculate the sum of squares of its individual digits. We find that in some cases, we arrive at the same number that we earlier came across during our calculation.`